src/test/java/sorinfoca/Tests/Tema8/TestPage/TestPages.java

```diff
@@ -0,0 +1,52 @@
+package sorinfoca.Tests.Tema8.TestPage;
+
+import org.testng.annotations.Test;
+import sorinfoca.Tests.Tema8.pages.*;
+import static org.testng.Assert.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+
+public class TestPages extends BaseTest {
+
+    @Test(groups = {"desktop", "mobile"})
+    public void testBasicAjaxPage() {
+        BasicAjaxPage basicAjaxPage = new BasicAjaxPage(driver);
+        basicAjaxPage.goToPage();
+        basicAjaxPage.selectOption("Server");
+        basicAjaxPage.selectLanguage("Java");
+        basicAjaxPage.clickCodeItInButton();
+        assertEquals(basicAjaxPage.getSelectedOption(),"23");
+    }
+
+    @Test(groups = {"desktop", "mobile"})
+    public void JavaScriptRedirectPage() {
+        JavaScriptRedirectPage javascriptRedirectPage = new JavaScriptRedirectPage(driver);
+        javascriptRedirectPage.goToPage();
+        javascriptRedirectPage.clickRedirectButton(0);
+        javascriptRedirectPage.switchToNewTab();
+        driver.close();
+    }
+
+    @Test(groups = {"desktop", "mobile"})
+    public void RefreshPage() {
+        RefreshPage refreshPage = new RefreshPage(driver);
+        refreshPage.goToPage();
+        assertTrue(refreshPage.isIdTimestamp());
```

Comment:
This test should have the following steps as per manual flow -> go to page, refresh page, assert that timestamp have changed after refresh. Please add the missing steps -> refresh page, assert that timestamp have changed after refresh. Thanks!